gnocchi-cli/src/main/scala/net/fnothaft/gnocchi/cli/RegressPhenotypes.scala

```diff
@@ -95,6 +97,9 @@ class RegressPhenotypesArgs extends Args4jBase {
 
   @Args4jOption(required = false, name = "-oneTwo", usage = "If cases are 1 and controls 2 instead of 0 and 1")
   var oneTwo = false
+
+  @Args4jOption(required = false, name = "-time", usage = "Prints out the timing information for RegressPhenotypes to commandline.")
```

Comment:
in ADAM we use `-print_metrics` for this, which comes from bdg-utils 

https://github.com/bigdatagenomics/utils/blob/master/utils-cli/src/main/scala/org/bdgenomics/utils/cli/Args4j.scala#L26

Or am I missing the use case here?